src/wizards/clusters.ts

```diff
@@ -16,8 +15,79 @@ const DEFAULT_STEPS = 4;
 
 export async function addClusterWizard(clusterSettings: ClusterSettings, explorer: KafkaExplorer): Promise<void> {
 
+    async function pickClusterProvider(): Promise<ClusterProvider | undefined> {
+        const providers = getClusterProviders();
+        if (providers.length === 1) {
+            // By default, it exists the default cluster provider 'Manual' from vscode-kafka
+            // to fill a cluster with a wizard, return it.
+            return providers[0];
+        }
+
+        const providerItems: QuickPickItem[] = providers
+            .map(provider => {
+                return { "label": provider.name };
+            });
+        const selected = (await window.showQuickPick(providerItems))?.label;
+        if (!selected) {
+            return;
+        }
+        return providers.find(provider => provider.name === selected);
+    }
+
+    // Pick the cluster provider which provides teh capability to return a list of cluster to add to the kafka explorer

```

Comment:
... the capability to return a list of clusters...

Reply:
fixed